flox-bash/lib/bootstrap.sh

```diff
@@ -25,12 +25,12 @@ function bootstrap() {
 	floxUserMetaRegistry get floxClientUUID >/dev/null || \
 		floxUserMetaRegistry set floxClientUUID $($_uuid)
 	floxClientUUID=$(floxUserMetaRegistry get floxClientUUID)
-	if [ -t 1 ] && [ -z "$PS1" ]; then
+	if [ -t 1 ] && [ -z "${PS1:-}" ]; then
```

Comment:
wouldnt the very correct way be something like 
```suggestion
	if [ -t 1 ] && [ -z "${PS1+empty}" ]; then
```